package.json

```diff
@@ -10,11 +10,11 @@
     "bootstrap": "^5.1.3",
     "popper.js": "^1.16.1",
     "turbolinks": "^5.2.0",
-    "webpack": "^4.46.0",
-    "webpack-cli": "^3.3.12"
+    "webpack": "4.46.0",
```

Comment:
Any particular reason to change the version constraints of these packages? 🤔

Reply:
We got an error when ran the app, the error shows something like webpacker compilation is not configured as true, and that we need to install weback, I do not remember the error specifically, but we found as a solution run the command: rails webpacker:install